src/models/disassemblyoutput.cpp

```diff
@@ -47,15 +53,18 @@ DisassemblyOutput::LinkedFunction extractLinkedFunction(const QString& disassemb
 }
 }
 
-static QVector<DisassemblyOutput::DisassemblyLine> objdumpParse(const QByteArray &output)
+static ObjectdumpOutput objdumpParse(const QByteArray& output)
 {
     QVector<DisassemblyOutput::DisassemblyLine> disassemblyLines;
 
     QTextStream stream(output);
     QString asmLine;
+    QString sourceFileName;
     // detect lines like:
     // 4f616: 84 c0 test %al,%al
```

Comment:
the comment needs to be updated now, as the below regex has changed?

Reply:
not really, the old regex also validated the hex representation of instructions, but this is not necessary

(note: the change was by accident)